labels-annotation-for-openshift.adoc

```diff
@@ -115,8 +115,8 @@ TIP: Labels must be applied to ALL resources, DeploymentConfigs, BuildConfigs, S
 
 | app.openshift.io/connects-to:
 | OPT
-| The component that this component is connecting to (e.g. a component making API calls to another `service` )
-| e.g. Component `front-end` has a `connects-to` which has a value `backend`
+| The `app.openshift.io/connects-to` contains the value of the label `app.kubernetes.io/instance` in the component being connected to.
+| e.g. Component `catalog` has a `connects-to` annotation which has a value `inventory`. 
```

Comment:
Shouldn't he example column, contain the example value for this label?

Reply:
Sorry, I might be missing something ?
 We have "e.g. Component `catalog` has a `connects-to` annotation which has a value `inventory`" in that column.

Reply:
What I meant was that from reading the document I thought that this column should contain example value for this label. Other rows have nice examples like (`v1.0.0` or `git://git.kernel.org/pub/scm/linux.git`). String `e.g. Component front-end catalog has a connects-to annotation which has a value backendinventory.` doesn't look like the correct value for `app.openshift.io/connects-to` label.